src/ticketandersencourse/model/Ticket.java

```diff
@@ -1,16 +1,22 @@
-package ticketandersencourse;
+package ticketandersencourse.model;
+
+import ticketandersencourse.enums.StadiumSector;
+import ticketandersencourse.interfaces.Printable;
+import ticketandersencourse.interfaces.Shareable;
+import ticketandersencourse.validation.Validator;
 
 import java.math.BigDecimal;
 import java.time.Instant;
+import java.util.List;
 import java.util.Objects;
 
-public class Ticket {
+public class Ticket implements Shareable, Printable {
     private String ID;
     private String concertHall;
     private int eventCode;
-    private long time;
+    public long time;
```

Comment:
All fields should be private if you need to mutate them somehow, create getters and setters and use them instead